Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -348,17 +351,40 @@ public class EOSSessionsManager : IEOSSubManager
         private SessionDetails JoiningSessionDetails = null;
         private ulong JoinedSessionIndex = 0;
 
+        /// <summary>
+        /// When joining a Session you did not create, the Session's name is not shared with you.
```

Comment:
nit: It would be clearer to say simply, "...the Session name..."
I would also rephrase to avoid use of any pronouns (ie "you").

Reply:
Noted! I am definitely awkward with pronouns in these comments, and it seems really easy to rephrase those away.